app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -30,17 +41,17 @@ def async_state_done
 
       def process_result(result)
         add_cost(:acuant_result) if result.to_h[:billed]
-        response = idv_result_to_form_response(result)
-        response.success?
       end
 
-      def async_state
+      def document_capture_session
         dcs_uuid = flow_session[verify_document_capture_session_uuid_key]
-        dcs = DocumentCaptureSession.find_by(uuid: dcs_uuid)
-        return ProofingDocumentCaptureSessionResult.none if dcs_uuid.nil?
-        return ProofingDocumentCaptureSessionResult.timed_out if dcs.nil?
+        @document_capture_session ||= DocumentCaptureSession.find_by(uuid: dcs_uuid)
```

Comment:
> this is not a good way to memoize an object that may be nil, since it'll do another DB query on a second check. could do:

Good call, thanks. Updated in 5b078da2c.